x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/application_added_to_google_workspace_domain.json

```diff
@@ -33,5 +33,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 8
+  "version": 10
```

Comment:
Some rules like this one have a version bump +1, while others were +2. Do you know why?

Reply:
I am not exactly sure why, but Justin informed me that is fine. As long as we do not see any sort of decrease.

Reply:
https://github.com/elastic/detection-rules/commit/f4c94af994a745f25a2322132b0b6e3e1d715d69